content/microservices/basics.md

```diff
@@ -232,6 +232,30 @@ The `options` property is an object with the same properties we saw in the `crea
 })
 ```
 
+Or alternatively using `registerAsync()` method in case if you want to pass configuration or doing any other async progress.
+
+```typescript
+@Module({
+  imports: [
+    ClientsModule.registerAsync([
+      {
+        imports: [ConfigModule],
+        name: 'MATH_SERVICE',
+        useFactory: async (configService: ConfigService) => ({
+          options: {
+            url: configService.URL,
```

Comment:
as `@nestjs/config` we have a provider called `ConfigService` people would probably think that this parameter is from there. And there's no `configService.URL` for that `ConfigService` from `@nestjs/config`
```suggestion
            url: configService.get<string>('url'),
```

Reply:
@micalevisk Thanks, done